modules/apps/remote-js-script/remote-js-script-api/src/main/java/com/liferay/remote/js/script/model/RemoteJsScriptEntryModel.java

```diff
@@ -323,6 +323,20 @@
 	 */
 	public void setUrl(String url);
 
+	/**
```

Comment:
You don't do this by handle, because it is using Service Builder. You need to add the new field to the `service.xml` file, then run `gradle buildService` and it will update everything for you. It is also a good idea to put the `service.xml` change in one commit, and the `gradle buildService` one in another so that it is easier to review.